pkg/types/userconfig/config_key.go

```diff
@@ -31,13 +31,17 @@ const (
 	TypeKey                   = "type"
 	PathKey                   = "path"
 	ModelKey                  = "model"
+	ModelsKey                 = "models"
 	PythonPathKey             = "python_path"
 	ImageKey                  = "image"
 	TensorFlowServingImageKey = "tensorflow_serving_image"
 	ConfigKey                 = "config"
 	EnvKey                    = "env"
 	SignatureKeyKey           = "signature_key"
 
+	// ModelResource
+	ModelsNameKey = "name"
```

Comment:
`NameKey` is already used for the API's name. If we ever want to change the model's key name, it could get a bit more complex since it would be shared. What do you think?